concordium-node/src/consensus_ffi/ffi.rs

```diff
@@ -249,6 +249,7 @@ type RegenesisFreeCallback = unsafe extern "C" fn(*const RwLock<Vec<BlockHash>>)
 extern "C" {
     pub fn startConsensus(
         max_block_size: u64,
+        block_construction_timeout: u64,
```

Comment:
We'll also want to add this parameter to the config structure.

This is in `concordium-node/concordium-node/src/configuration.rs`, specifically the BakerConfig structure (next to `maximum_block_size`.